README.md

```diff
@@ -64,17 +70,19 @@ deblur workflow --help
 
 Positive and Negative Filtering
 ===============================
-By default, deblur uses positive filtering, keeping only 16S sequences (based on homology to [Greengenes](http://greengenes.secondgenome.com/) [88% representative set](deblur/support_files/88_otus.fasta)). For example:
+Deblur uses two types of filtering on the sequences:
 
-```
-deblur workflow --seqs-fp all_samples.fna --output-dir output
-```
+- negative mode - removes [known artifact sequences](deblur/support_files/artifacts.fa) (i.e. sequences aligning to PhiX or Adapter with >=95% identity and coverage).
 
-Negative filtering can be selected using the '-n' flag. This causes deblurring to keep all sequences except for [known artifact sequences](deblur/support_files/artifacts.fa) (i.e. PhiX and Adapter sequences), so other non-16S sequences are retained. For example:
+- positive mode - keeps only sequences similar to [known 16S sequences](deblur/support_files/88_otus.fasta) (e-value<=10 for Greengenes 88% rep. set). Note that deblur also outputs a biom table without this positive filtering step.
```

Comment:
Yup. sortmerna e<=10 is what Jenya recommended, and also works well in the cases we tested.